src/sentry_backend.h

```diff
@@ -20,6 +20,7 @@ typedef struct sentry_backend_s {
     void (*except_func)(
         struct sentry_backend_s *, const struct sentry_ucontext_s *);
     void (*flush_scope_func)(struct sentry_backend_s *);
+    // NOTE: the breadcrumb is borrowed and does not need to be `decref`-d.
```

Comment:
Not going to lie, this is ambiguous to me.  "is borrowed" i understand that the caller retains ownership, so if the caller owns are they not responsible for decref-ing?